cfpb/cfgov-refresh
# Update to FOIA FAQ Page

Replacing the placeholder activity feed with an actual activity feed on the FOIA FAQ page

## Changes

- Replacing the placeholder activity feed with an actual activity feed on the FOIA FAQ page

## Testing

- Visit /initiatives/foia-faqs/
- Scroll to see the new activity feed

## Review

- @jimmynotjim 
- @sebworks 
- @anselmbradford 

## Preview

![image](https://cloud.githubusercontent.com/assets/1860176/7870370/dd33712c-0556-11e5-808d-5926643960e3.png)

[Preview this PR without the whitespace changes](?w=0)

## Notes

- Confirmed that this was supposed to go there during Tuesday's Team Review
- I have the content to replace the Lorem Ipsum on the actual expandable, but I'm leaving it as is to clear that we know when we're using placeholder and when we're using the information from the CMS.
